package.yaml

```diff
@@ -1,14 +1,15 @@
-name: haskoin-core
-version: 0.21.2
-synopsis: Bitcoin & Bitcoin Cash library for Haskell
+name: bitcoin
+version: 0.1.0
+synopsis: Bitcoin library for Haskell
 description: Please see the README on GitHub at <https://github.com/haskoin/haskoin-core#readme>
```

Comment:
The URLs need to be updated (here and a few below)

Reply:
ah nice good catch